pkg/adaptor/proxy/service.go

```diff
@@ -169,6 +181,17 @@ func (s *proxyService) CreateContainer(ctx context.Context, req *pb.CreateContai
 	return res, err
 }
 
+func isNodePublishVolumeTargetPath(volumePath, directVolumesDir string) bool {
+	if !strings.Contains(volumePath, csiPluginEscapeQualifiedName) {
```

Comment:
Can the following code work?

```
if filepath.Base(volumepath) != csiPluginEscapeQualifiedName {
```

Reply:
No. volumepath value may looks like: `/var/lib/kubelet/pods/cc962e00-5575-4ca4-a651-afb5e1ef5d90/volumes/kubernetes.io~csi/pvc-c07c4faf-790f-4c1b-b626-6202937967f0/mount`, csiPluginEscapeQualifiedName value is `kubernetes.io~csi`

Reply:
Thank you for the explanation.

So, how about this one?  This causes less false-positives.

```
if !strings.Contains(filepath.Clean(path), "/volumes/"+csiPluginEscapeQualifiedName+"/") {
```

Reply:
@yoheiueda , sure. Made the changes and add test code